### DISCUSSION TITLE
Consigli tunnel Cloudflare

### DISCUSSION THREAD

**Discussion by avvalex** [indomus/forum#2019] [GENERALE - Discussioni generiche]
Buongiorno ragazzi, vorrei cimentarmi nel passaggio da DuckDNS+Letsencrypt al collegamento da remoto mediante tunnel Cloudflare+Dominio Cloudflare (o altro... ma starei su Cloudflare, vorrei un dominio a una decina di euro che ogni anno si rinnovi in automatico), il tutto sotto Docker.
Poiché sono un po' acerbo e vorrei evitare errori grossolani, c'è qualcosa a cui dovrei prestare particolare attenzione? Errori da evitare?
Espongo solo HA oppure tutto? Controindicazioni?
In giro si trovano guide con HA supervised, ma mi fido poco (più che di quello che dicono di fare, magari di qualcosa che non dicono).
Il tutto in attesa di una bella guida di Indomus.... :-)

**Comment by 50494554524F** [comment]
stavo sperimentando e mi son ricordato di questo 3d... 

per fare le cose semplicissime senza configurare una vpn puoi usare l'addon per tor. su smartphone userai orbot e quindi nell'app homeassistant metterai l'indirizzo onion che ti fornisce l'addon, finito... da pc puoi usare direttamente tor browser. ci ho messo più a leggere di tor che non avevo mai usato che a configurare questo...

visto che è difficile ricordarsi l'indirizzo onion puoi registrare un dominio gratuito .tk (eventualmente configurando domini di terzo livello con forwarddomain.net o simili)... ovviamente prendendo un dominio a pagamento  o un redirect su un qualsiasi hosting gratuito con dominio facile da ricordare non c'è bisogno di questi giri :D
